cartridges/int_bolt_sfra_ppc/cartridge/client/default/js/boltProductPageButton.js

```diff
@@ -0,0 +1,295 @@
+'use strict';
+
+var ppcButtonClass = 'bolt-product-checkout-button';
+// isProductPageCheckoutButtonVisible is used to track the visibility of the PPC button
+var ppcButtonVisible;
+// ppcCart contains BoltCart which is obtained by calling BoltCheckout.configureProductCheckout
+var ppcCart;
+// setupPPCButtonPoller is the ID of the timer that is used to show the Bolt PPC button.
+var setupPPCButtonPoller;
+
+var successRedirect = $('#successRedirect').val();
+var sfccData;
+var callbacks = {
+    check: function () {
+        console.log('Inside check function');
```

Comment:
I imagine this is a debug print, should this be deleted?

Reply:
I think it's OK to keep the log.